### PR TITLE
adding missing params

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/10024

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: skype/skype-ps/skype/Set-CsTenantNetworkSite.md [hunk 1]
```diff
@@ -251,6 +253,36 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -EmergencyCallingPolicy
+This parameter is used to assign a custom emergency calling policy to a network site. For more information see [Assign a custom emergency calling policy to a network site](/microsoftteams/manage-emergency-calling-policies#assign-a-custom-emergency-calling-policy-to-a-network-site).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTenantNetworkSite.md:272]:
```suggestion
This parameter is used to assign a custom emergency calling policy to a network site. For more information, see [Assign a custom emergency calling policy to a network site](/microsoftteams/manage-emergency-calling-policies#assign-a-custom-emergency-calling-policy-to-a-network-site).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by tseward — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**